front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleConfigV2.tsx

```diff
@@ -120,11 +71,6 @@ const ImportTrainScheduleConfigV2 = ({
   }
 
   async function updateTrainSchedules(importedTrainSchedules: ImportedTrainSchedule[]) {
```

Comment:
The function is not async anymore, you can remove 'async' as well as the 'await' l105
```suggestion
  function updateTrainSchedules(importedTrainSchedules: ImportedTrainSchedule[]) {
```